pulsar-broker/src/main/java/org/apache/pulsar/broker/service/ServerCnx.java

```diff
@@ -1193,7 +1193,7 @@ protected void handleProducer(final CommandProducer cmdProducer) {
                                 topic.addProducer(producer, producerQueuedFuture).thenAccept(newTopicEpoch -> {
                                     if (isActive()) {
                                         if (producerFuture.complete(producer)) {
-                                            log.info("[{}] Created new producer: {}", remoteAddress, producer);
+                                            log.debug("[{}] Created new producer: {}", remoteAddress, producer);
```

Comment:
I don't think these are redundant as in they're logging when a producer is effectively active.

Reply:
@merlimat I see your point, here we are sure that the producer is actually "active".
What about reducing log level of "Creating producer. " instead of this line ?

the problem is the amount of logs we see when the system is working properly, we have 2 or 3 lines per producer, and this is a certain waste of log resources. 
the problem is more evident in case of short lived producers (like health checks)